specification/sources/chapters/instance.adoc

```diff
@@ -181,7 +181,15 @@ different results if a pname:layerName is available in one call but not in
 another.
 The extensions supported by a layer may also change between two calls, e.g.
 if the layer implementation is replaced by a different version between those
-calls.
+calls, or if an `XrInstance` is created or destroyed between those calls.
+
+This function may: be called before an instance has been created;
+implementations must: not assume an instance exists.
+
+When applications invoke this function, the OpenXR loader's implementation
+is used, which enumerates the registered runtime and API layer manifests;
+this function must: still be implemented by API layers and runtimes, as it may
+be invoked by API layers.
```

Comment:
I think I'd also like to see:
```suggestion
When applications invoke this function, the OpenXR loader's implementation
is used, which enumerates the registered runtime and API layer manifests;
this function must: still be implemented by API layers and runtimes, as it may
be invoked by API layers, and must: return the same information a the manifest
```

Reply:
Would that belong here, or in the loader docs over on https://github.com/KhronosGroup/OpenXR-SDK-Source/pull/490/files ?

Reply:
Good question, I think it belongs here because it's specifying the behaviour of the function rather than the detail of the manifest. It probably should have an accompanying note in the loader docs that mirrors it form the manifest persepective.

Reply:
The problem with adding it here is that the main spec doesn't know about the loader - or implementation details of a potential loader like the manifest. It does sound good to add it to the loader spec though.